index.html

```diff
@@ -0,0 +1,184 @@
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&family=Roboto:wght@100&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header class="main-head">
+    <nav class="toolbar">
+      <div class="top-toolbar">
+        <div><h2>Smart City</h2></div>
+        <a href=""><i class=" fas fa-bars fa-2x"></i></a>
+      </div>
+    </nav>  
+  </header>
+  <section class="headline">
+    <div class="section-head">
+      <div class="title-bkg">
+        <h1>Ademola Adebayo</h1>
+        <h3>Software Developer</h3>
+      </div>
+      <div class="head-text">
+        <p>I can help you build a product,<br> feature or website Look through<br> some of my work and experience! If<br> you like what you see and have 
+        a<br> project you need coded, don’t<br> hestiate to contact me.</p>
+      </div>
+      <hr> 
+    </div>
+    <div class="lang-framework">
+      <div>
+        <p class="lang-col">Language</p>
+        <ul>
+          <li>Javascript</li>
+          <li>HTML</li>
+          <li>Ruby</li>
+          <li>CSS</li>  
+        </ul>
+      </div>
+      <div class="side-2">
+        <p class="lang-col">Frameworks</p>
+        <ul>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>React</li>
+          <li>Redux</li>
+          <li>State</li>
+        </ul>
+      </div>  
+    </div>
+    <div class="skills">
+      <ul>
+        <li class="lang-col">Skills</li>
+        <li>Codekit</li>
+        <li>Github</li>
+        <li>Codelien</li>
+        <li>Terminal</li>
+        <li>Replit</li>
+      </ul>  
+    </div>
+    <hr>
+    <div class="social">
+      <ul>
+        <li><a href="#"><i class="fab fa-github fa-2x"></i></a></li>
+        <li><a href="#"><i class="fab fa-facebook fa-2x"></i></a></li>
+        <li><a href="#"><i class="fab fa-instagram fa-2x"></i></a></li>
+        <li><a href="#"><i class="fab fa-twitter fa-2x"></i></a></li>
+      </ul>
+    </div>
+    <div class="dn-page">
+      <a href="#"><i class="fas fa-arrow-down fa-2x"></i></a>
+      <a href="#"><p>More</p></a>
+    </div>  
+  </section>
+  <!-- Work-Section -->
+  <section class="work-grid">
+    <div class="recent">
+      <h3>MY RECENT<br>WORK</h3>
+    </div>
+    <div class="sub-grid">
+      <div class="box">
+        <input type="text" placeholder="">
+      </div>
+      <div class="story-title">
+        <div>
+          <h2>Multi-Post Stories</h2>
+          <p>A daily selection of privately<br> personalized read; no accounts<br> or sign-ups required, has been
+          the<br> industry's standard dummy text<br> ever since the 1500s, when an<br> unknown printer took a
+          took a standard<br> dummy text</p>
+        </div>
+        <div>
+          <ul>
+            <li>css</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
```

Comment:
- [ ] [OPTIONAL] You have done a great job implementing this design, however, kindly remove the last stroke after `ruby` in the list of languages. Implement this everywhere this has been repeated in order for your design to look like the template provided.
### Your Implementation
![Screenshot from 2022-03-21 14-20-15](https://user-images.githubusercontent.com/58692590/159271614-a2d0c4f8-0039-4d86-88b3-2447728f0c7f.png)

### Template Provided
![Screenshot from 2022-03-21 14-20-23](https://user-images.githubusercontent.com/58692590/159271698-cadac78c-9d0a-4ad4-89a4-d62d2c2b40c3.png)